.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: CI
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  MacOS:
+    runs-on: macos-latest
+    defaults:
+      run:
+        shell: bash
+    env:
+      PYTHON_VERSION: '3.7'
+      MNE_LOGGING_LEVEL: 'warning'
+      OPENBLAS_NUM_THREADS: '1'
+      PYTHONUNBUFFERED: '1'
+      CONDA_ENV: 'environment.yml'
+      TRAVIS_OS_NAME: 'osx'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v1
+        env:
+          # Increase this value to reset cache if environment.yml has not changed
+          CACHE_NUMBER: 0
+        with:
+          path: ~/conda_pkgs_dir
+          key: ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{ hashFiles('environment.yml') }}
+        name: 'Cache conda'
+      - uses: goanpeca/setup-miniconda@v1
+        with:
+          activate-environment: 'mne'
+          python-version: ${{ env.PYTHON_VERSION }}
+          environment-file: ${{ env.CONDA_ENV }}
+          use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
+        name: 'Setup conda'
+      - shell: bash -l {0}
```

Comment:
It seems like it would be safer to make all of these `bash -el` so that an error on any subcommand will exit